util/cron/nightly_email_file.pm

```diff
@@ -8,22 +8,26 @@ use nightlysubs;
 sub writeFile{
     $num_args = @_;
 
-    if ($num_args != 18) {
+    if ($num_args != 17) {
         print "usage: nightly_email.pm \$status \$rawsummary \$sortedsummary \n";
```

Comment:
Also need to update filename here

Reply:
Doh, more pushups.    Just pushed an update to update the usage string.